setup.py

```diff
@@ -139,13 +139,14 @@ def finalize_options(self):
     ],
     extras_require={
         'docs': [
-            'sphinxcontrib-bibtex',
-            'sphinxcontrib-websupport',
-            'pydata-sphinx-theme'
+            'Sphinx>=3.0,<=4.0',
+            'pydata-sphinx-theme',
```

Comment:
Will this play nicely with readthedocs? Have no experience with that

Reply:
Should, all versions of Sphinx are supported